client/src/components/Button/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import classes from './style.css'
+import {
+  faArrowLeft,
+  faArrowRight,
+  faCheckCircle
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+export default function Button(props) {
+  return (
+        <button className={classes.button} style={props.style} onClick={props.onClick}>
```

Comment:
you can delete the style attribute, no need for it
just add a class name  for the button 
on is custom and the other will depend on where the button used

Reply:
Done